apps/wallet-mobile/src/yoroi-wallets/cardano/shelley/ShelleyWallet.ts

```diff
@@ -453,6 +454,32 @@ export const makeShelleyWallet = (constants: typeof MAINNET | typeof TESTNET) =>
       return stakingKey
     }
 
+    public async signRawTx(txHex: string, pKeys: PrivateKey[]) {
```

Comment:
Do we need implement this function in two places ? Can we move it into separate file ?